packages/react-component-library/src/components/Modal/Modal.tsx

```diff
@@ -9,6 +9,8 @@ import { useOpenClose } from '../../hooks/useOpenClose'
 import { getId } from '../../helpers'
 
 export interface ModalProps extends ComponentWithClass {
+  titleId?: string
```

Comment:
It could be easier to read if these were alphabetical.

Reply:
These have now all been alphabetised.